pengwin-setup.d/pythonpi.sh

```diff
@@ -10,34 +10,33 @@ if (whiptail --title "PYTHON" --yesno "Would you like to download and install Py
     bash pyenv-installer
 
     echo "inserting default scripts"
-    
+
     if [ -f ${HOME}/.bashrc ]; then
      	echo "export PATH=\"${HOME}/.pyenv/bin:\$PATH\"" >> ${HOME}/.bashrc
       	echo "eval \"\$(pyenv init -)\"" >> ${HOME}/.bashrc
       	echo "eval \"\$(pyenv virtualenv-init -)\"" >> ${HOME}/.bashrc
-	source ${HOME}/.bashrc
     fi
 
     if [ -f ${HOME}/.zshrc ]; then
     	echo "export PATH=\"${HOME}/.pyenv/bin:\$PATH\"" >> ${HOME}/.zshrc
       	echo "eval \"\$(pyenv init -)\"" >> ${HOME}/.zshrc
       	echo "eval \"\$(pyenv virtualenv-init -)\"" >> ${HOME}/.zshrc
-	source ${HOME}/.zshrc
     fi
 
     if [ -d ${HOME}/.config/fish ]; then
 	echo "set -x PATH \"${HOME}/.pyenv/bin\" \$PATH" >> ${HOME}/.config/fish/config.fish
       	echo 'status --is-interactive; and . (pyenv init -|psub)'  >> ${HOME}/.config/fish/config.fish
       	echo 'status --is-interactive; and . (pyenv virtualenv-init -|psub)' >> ${HOME}/.config/fish/config.fish
-	source ${HOME}/.config/fish/config.fish	
     fi
 
     echo "installing Python 3.7"
+    export PATH="${HOME}/.pyenv/bin:$PATH"
+    eval "$(pyenv init -)"
+    eval "$(pyenv virtualenv-init -"
```

Comment:
The issue of this is that this doesn't work in fish. I have tested this before and because of this reason, I remove it and suggest user just to restart the shell
![image](https://user-images.githubusercontent.com/15316889/57079851-d5d06c80-6d23-11e9-868a-545d049cb574.png)

Reply:
I know this doesn't work in fish, and I'm not sure if it will in zsh either. But the point I was making was that since our script is run in Bash, it will work in setting the environment variables correctly just to install the latest version of Python with pyenv.

Reply:
So maybe for this part, the `eval` is not needed? because if we just need to install it, we can just remove this part and using `${HOME}/.pyenv/bin/pyenv <command>` when installing python and ask users to restart the shell.

Reply:
considering the internal complexity of pyenv I think it might be safest to just leave these in place, it's during an install script (not UI or menu functions) and adds very very little time to the script, especially compared to the time required for `apt-get install ` or even `pyenv install `.

Reply:
Yeah, that makes sense. Maybe adding a prompt that tells people to restart their prompt to activate at the end?